polkadot-parachains/integritee-runtime/src/lib.rs

```diff
@@ -409,14 +392,17 @@ impl pallet_proxy::Config for Runtime {
 
 parameter_types! {
 	pub const MinVestedTransfer: Balance = 1 * TEER;
+		pub UnvestedFundsAllowedWithdrawReasons: WithdrawReasons =
+			WithdrawReasons::except(WithdrawReasons::TRANSFER | WithdrawReasons::RESERVE);
```

Comment:
Todo: double-check this config. It is taken from https://github.com/integritee-network/integritee-node/pull/159